warehouse/static/sass/components/_project-description.scss

```diff
@@ -2,12 +2,12 @@
   Styles the main description panel for an individual package.
```

Comment:
While we are here, can we also change this to say "Styles the main description panel for an individual project"